### PR TITLE
adding parallel example in readme

### PR DESCRIPTION
**Pull request by philbucher**
feel free to modify if you like :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,16 +37,67 @@ int main( )
...
+        points,       // coordinates of the points
+        connectivity, // connectivities of the cells
+        offsets,      // offsets between the cell points
+        types         // types of the cells, see "https://vtk.org/wp-content/uploads/2015/04/file-formats.pdf"
+    };
 
     // Write data to .vtu file using raw binary appended format
-    vtu11::write( "test.vtu", mesh, { pointDataSet }, { cellDataSet }, 
-                  vtu11::RawBinaryAppendedWriter { } );
+    vtu11::write(
```
Comment (phmkopp) [inline README.md:50]: I don't like this format but if you really want we can keep it.

Comment (philbucher) [reply README.md:50]: initially I wanted to add explanations to the arguments (like I did in the parallel example) but then I didn't
I can revert, but then it is inconsistent with the parallel example :D

**Comment by phmkopp** [comment]
I'll try to make something compilable out of the parallel example.

**Comment by philbucher** [comment]
closing as contained in #27

**Pull request closed without merging**